Packs/Automox/Integrations/Automox/README.md

```diff
@@ -1,5 +1,655 @@
-This README contains the full documentation for your integration.
+Use the Automox integration to create device groups, update devices, run policies, and remediate vulnerabilities of devices in your Automox instance.
```

Comment:
```suggestion
Use the Automox integration to create device groups, update devices, run policies, and remediate vulnerabilities of devices through the Automox platform.
```

Reply:
OOh, I'm saving this verbiage in my pocket. Committed.